addon/components/citation-plugin/citations/decision-preview.ts

```diff
@@ -0,0 +1,19 @@
+import Component from '@glimmer/component';
+import {
+  isBesluitLegalDocument,
+  LegalDocument,
+} from '@lblod/ember-rdfa-editor-lblod-plugins/plugins/citation-plugin/utils/legal-documents';
+
+interface Args {
+  decision: LegalDocument;
+}
+
+export default class DecisionPreviewComponent extends Component<Args> {
+  get isDecision() {
+    return isBesluitLegalDocument(this.args.decision);
+  }
```

Comment:
example of where the term "decision" is used in two different contexts. It is a `DecisionPreview` component, so you'd expect `isDecision` would always be true. But they both refer to a different type of "decision".